Cython/Utility/ImportExport.c

```diff
@@ -334,12 +335,23 @@ static int ${import_star}(PyObject* m) {
     if (__Pyx_import_all_from(locals, m) < 0) goto bad;
     list = PyDict_Items(locals);        if (!list) goto bad;
 
+#if CYTHON_COMPILING_IN_LIMITED_API
+    for(i=0; i<PyList_Size(list); i++) {
+        name = PyTuple_Get_Item(PyList_Get_Item(list, i), 0); if (!name) goto bad;
+        item = PyTuple_Get_Item(PyList_Get_Item(list, i), 1); if (!item) goto bad;
```

Comment:
Should we add `__Pyx_Py[Tuple|List]_GET_ITEM` to ModuleSetupCode.c? I imagine this won't be the last time we need this kind of conditional otherwise.

Reply:
Yes - it looks like we already have `SET_ITEM` so I think that'd make sense.